src/it/big-artifact/postbuild.groovy

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2020 Markus Karg
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import java.io.FileNotFoundException
+import java.nio.file.Files
+
+def expectedName = "big-artifact-1.1.1.dat.asc"
+def target = basedir.toPath().resolve( "target" )
+def expectedPath = target.resolve( expectedName )
+
+if ( Files.notExists( expectedPath ) )
+{
+    throw new FileNotFoundException( "Expected file is missing: " + expectedPath )
+}
```

Comment:
it can be:
```
assert Files.exists(expectedPath)
```

Reply:
Thank you for this idea! I didn't know what!